### DISCUSSION TITLE
TubeMod 2.0 - Extension Redesign, Safari and Custom Stores Support, Themes and more

### DISCUSSION THREAD

**Discussion by Pedro-Gregorio** [Pedro-Gregorio/TubeMod#109] [Q&A]
Hello everyone, hope you're all doing well! Let's talk about the 2.0.0 version of TubeMod.

1st, I'm planning on redesigning the extension for an upcoming release, probably before the end of the year. On version 2.0.0 of TubeMod, my goal is to have a different overall design for the extension and properly organizing the way you interact with the extension. We have 71 options right now, and more are coming. Stacking them all on the same place doesn't feel right. TubeMod was created to mess with it once, and forget it's there, but that doesn't mean that experience can't be better 😄 

2nd, Safari is coming, as well as integrations on Edge Store, Opera Store, and so on. Feel free to recommend more down here 👍 

3rd, Themes are also planned for this next version. After 1.10.0, I'll probably start looking into this, and dropping small changes to 1.x versions until the next one is fully ready to gather more feedback, as I need more opinions on what's really needed here! Also, TubeMod is an extension, could TubeTheme be the next one? How would you feel about this separation?

4th, anything else you remember, please let me know! We've crossed 12k users this weekend, and I still feel like I can push this a little bit more 😅 

With that being said, please drop your suggestions here!

**Comment by Pedro-Gregorio** [comment]
One more thing people asked for in the past and I still haven't got a good idea on how to do: moving the extension around to see changes happening... I may create a settings page to tackle both this and the 1st problem I mentioned.

**Reply by aaronhlaw** [reply]
Yea I was thinking of doing making some pull requests to help the project out. Should I start contributing or wait until this gets done?
